src/DbTests/SQL/MenyColumns.sql

```diff
@@ -0,0 +1,602 @@
+﻿SELECT 
+CASE WHEN 1=1 THEN NULL ELSE '' END AS Column1,
```

Comment:
Why all the `CASE WHEN 1=1 THEN NULL ELSE '' END`? Why not just

```sql
NULL AS Column1
NULL AS Column2
...
```

Reply:
I did this to make it a nullable string but type doesn't really matter.  This makes it a nullable int.